package/yast2-python-bindings.spec

```diff
@@ -33,42 +44,93 @@ BuildRequires:  gcc-c++
 BuildRequires:  libtool
 BuildRequires:  libyui-devel
 BuildRequires:  make
+BuildRequires:  python
+BuildRequires:  python-devel
+%if %{with_python3}
 BuildRequires:  python3
 BuildRequires:  python3-devel
+%endif
 BuildRequires:  swig
 BuildRequires:  yast2-core-devel
 BuildRequires:  yast2-ycp-ui-bindings
 BuildRequires:  yast2-ycp-ui-bindings-devel
-
-Requires:       python3
+Requires:       python
 Requires:       yast2-core
 Requires:       yast2-ycp-ui-bindings
+Obsoletes:      yast2-python-bindings < 4.0.1
+Conflicts:      yast2-python3-bindings
 
-Summary:        Python bindings for the YaST platform
-License:        GPL-2.0
+%description -n yast2-python-bindings
+The bindings allow YaST modules to be written using the Python language
+and also Python scripts can use YaST agents, APIs and modules.
+
+%if %{with_python3}
+%package -n yast2-python3-bindings
+Summary:        Python3 bindings for the YaST platform
 Group:          System/YaST
+Requires:       python3
+Requires:       yast2-core
+Requires:       yast2-ycp-ui-bindings
+Obsoletes:      yast2-python-bindings == 4.0.1
```

Comment:
These obsoletes are due to version 4.0.1 being accepted in SLE 15 (but rejected in openSUSE). yast2-python-bindings version 4.0.1 (in SLE15) is the python3 version of the package. Versions less than 4.0.1 are python2. So I have to handle just the upgrade of that package version scenario.